src/DotNetCore.CAP.RabbitMQ/IConnectionChannelPool.Default.cs

```diff
@@ -108,7 +111,7 @@ public virtual IModel Rent()
 
         public virtual bool Return(IModel connection)
         {
-            if (Interlocked.Increment(ref _count) <= _maxSize)
+             if (connection.IsOpen && Interlocked.Increment(ref _count) <= _maxSize)
```

Comment:
I think we need add the check of the model connection to the `IConnectionChannelPool.Return(IModel connection)` method , because  the `public virtual bool Return(IModel connection)` can be override by user, It might be missed.

Reply:
Not sure what's your mean of adding dd the check of the model connection to the IConnectionChannelPool.Return(IModel connection), it's an interface definition. 
For your concern, I think the developer who override the method should take this responsibility to ensure not return the unhealthy connection like what we do here :).

Reply:
we need keep the program robustness,  If user return a bad connection ,we should not add it to the connection pool,  so I think this check is necessary

Reply:
Please ignore my PR. Just noticed the latest RabbitMQ.Client has already support auto recovery  feature and by default it turns on. Please refer to the ConnectionFactyory class
public bool AutomaticRecoveryEnabled { get; set;} = true;

We used the old version (V3.x) due to compatibility. 
Sorry for the trouble and thanks for your time for reviewing my PR.